src/models/QueryStateModel.ts

```diff
@@ -46,6 +47,7 @@ export class QueryStateModel extends Model {
     hd: '',
     hq: '',
     sort: '',
+    layout: '',
```

Comment:
I think this should be 'list', meaning the list display is the default state.

Then, we need to add similar logic that tab and sort that we have in the SearchInterface, that will always fallback to defaultAttribute if something impossible/illogical is set.

Look for the `handlePreprocessQueryStateModel` method.